packages/react/src/core/ContextMenu.tsx

```diff
@@ -180,7 +182,9 @@ function Dialog({ title, onClose, children }: { title: string; onClose: () => vo
 
             <div className="my-4">{children}</div>
 
-            <RadixDialog.Close className="px-3 py-1 border border-tk-border-primary rounded">OK</RadixDialog.Close>
+            <RadixDialog.Close asChild>
+              <Button>OK</Button>
```

Comment:
How about we make it a bit larger? Wdyt?

```suggestion
              <Button className="min-w-20 justify-center">OK</Button>
```

Oh also we probably need `OK` to be `i18n?.confirmationText || DEFAULT_LOCALIZATION.confirmationText`